CONTRIBUTING.md

```diff
@@ -0,0 +1,3 @@
+We welcome your contributions to new features, fixes, or updates, large or small; we are always thrilled to receive pull requests, and do our best to process them as fast as we can.
+
+Before you start to code, we recommend discussing your plans through a [GitHub issue](https://github.com/NYPL-Simplified/circulation/issues/new), especially for more ambitious contributions. This gives other contributors a chance to point you in the right direction, give you feedback on your design, and help you find out if someone else is working on the same thing.
```

Comment:
It would be better to remove this than move it, since it doesn't accurately represent how we do development right now.

Reply:
If we want to make changes to the language for accepting open source contributions, that seems appropriate to better reflect our expectations. Since this is an open source repo, anyone can still submit a PR, even though the NYPL has its own roadmap - maybe even indicating that and a few extra notes about how we plan to triage contributions would be enough.